crates/daemon/src/wallet.rs

```diff
@@ -512,10 +579,42 @@ mod tests {
                     time_to_lock,
                 },
                 blockchain_client: (),
+                db: None,
             })
         }
     }
 
+    impl Actor<ElectrumBlockchain, Tree> {
+        pub async fn spawn_test(
```

Comment:
I'm not sure this is a good idea: this will run an online test against testnet using public infrastructure. 
Besides that it can have random side effects if the server is down, it might slow down our test executions. 

Do you think we can use our [test wallet](https://github.com/itchysats/itchysats/blob/master/crates/bdk-ext/src/lib.rs#L14) for this?

Reply:
well.. we could do that, but I have the feeling we would very quickly end up with testing only test code.

Reply:
@bonomat have a look at 01f62ef334e28a966bc975e724bba05c1beca2a8. It uses the test wallet now, and does not sync with electrum for balance, but I guess checking on the restored address is sufficient.